### PR TITLE
Latest rule engine upgrade for general rule

### PR DESCRIPTION
**Pull request by titame**
Hello Team,

This branch has changes in rule engine for sending mail for top 5 contributors list and last 24 hours activities  based on general rule set in rule engine.
Please review and let me know if any changes are required.

Thanks,
Tushar Titame

### CODE REVIEW

File: app/assets/javascripts/active_admin.js [hunk 1]
```diff
@@ -1 +1,2 @@
 #= require active_admin/base
+#= require rule_engine
```
Comment (rohandaxini) [inline app/assets/javascripts/active_admin.js]: there are multiple comments for issues

Comment (rohandaxini) [inline app/assets/javascripts/active_admin.js]: @titame I hope we are learning from these hound warnings and u took notes very first time itself from your previous review ?

**Comment by titame** [comment]
Yes I took note of it @rohandaxini

File: app/assets/javascripts/active_admin.js (hunk 1 above)
Comment (titame) [inline app/assets/javascripts/active_admin.js]: Yes I took notes of this and previous warnings @rohandaxini

Comment (rohandaxini) [inline app/assets/javascripts/active_admin.js]: @titame but you are repeating same mistakes, despite taking notes ? From hound comments it seems that we are repeating mistakes and not taking care of warnings in advance.

**Pull request merged**